docs/serialization.md

```diff
@@ -250,6 +250,27 @@ It's represented in JSON to:
 In a similar way to a record type, undefined fields in a payload are ignored
 by deserializer.
 
+If '@external-tag' annotation is given on tag below example
+
+     union name =
+         @external-tag
+         east-asian-name ( text family-name, text given-name )
+         | ...
+         ;
+
+It's representated in JSON to:
+
+     {
+        {
+            "east-asian-name": {
+                    "_type": "name",
+                    "_tag": "east-asian-name",
```

Comment:
This field might be not necessary, because it already presented above.

Reply:
And, I think `@external-tag` should be associated on the whole union, not on its tag. If not, it loses the advantage that can be decide which tag should be parsed at once.